### PR TITLE
New

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -24,14 +24,24 @@ Learn more about what is new this release:
...
+
+----
+https://grafana-open-cluster-management-observability.{OPENSHIFT_INGRESS_DOMAIN}
+----
+
+See link:../observability/design_grafana.adoc#console-overview[Designing your Grafana dashboard] for more information. 
+
+Learn more about the console in the link:../console/console.adoc#console-overview[Console overview].
 
 [#observability-whats-new]
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: Seems like this section should be removed and only web console should be updated?

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -24,14 +24,24 @@ Learn more about what is new this release:
 [#web-console-whats-new]
 == Web console
 
-Learn more in the link:../console/console.adoc#console-overview[Console overview].
+You can now access Grafana through the [ocp-short] route with the following URL:
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: use curly brackets for conref

**Review by dockerymick — CHANGES_REQUESTED** [review]
Verified links and left comments

File: release_notes/whats_new.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/whats_new.adoc]: ugh

File: release_notes/whats_new.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/whats_new.adoc]: Right, @dockerymick -- I asked about this on Slack so if you have no updates for this section, I will remove.

Comment (dockerymick) [reply release_notes/whats_new.adoc]: Sounds good, yea the current entry is related to observability, but i understand why we would keep it in Web console

**Review by dockerymick — APPROVED** [review]
looks good, thanks for the update!

**Pull request merged**